### ISSUE TITLE
myQ API: Unable to authenticate. Will retry later.

### ISSUE THREAD

**Issue by DNSGeek** [hjdhjd/homebridge-myq#201]
**Describe The Problem:**

Starting a few days ago, I lost connectivity to my myQ opener. I do not get open/close notifications, the Home app always shows the door as closed and is unable to open/close the door.

I can use the myQ app on my iPhone, that is able to open/close the door and give status. I then went and changed my password, changing it in both the home bridge config and the myQ app. Restarted home bridge.

myQ can still access my door, home bridge cannot.

**To Reproduce:**

**Logs:**

Over and over and over.
```
[16/12/2020, 21:00:01] [myQ] myQ API: Connection has been reset.
[16/12/2020, 21:00:01] [myQ] myQ API: Unable to authenticate. Will retry later.
[16/12/2020, 21:00:04] [myQ] myQ API: Connection has been reset.
[16/12/2020, 21:00:04] [myQ] myQ API: Unable to authenticate. Will retry later.
[16/12/2020, 21:00:07] [myQ] myQ API: Connection has been reset.
[16/12/2020, 21:00:07] [myQ] myQ API: Unable to authenticate. Will retry later.
[16/12/2020, 21:00:10] [myQ] myQ API: Connection has been reset.
[16/12/2020, 21:00:10] [myQ] myQ API: Unable to authenticate. Will retry later.
[16/12/2020, 21:00:14] [myQ] myQ API: Connection has been reset.
[16/12/2020, 21:00:14] [myQ] myQ API: Unable to authenticate. Will retry later.
```

**Homebridge Configuration:**

```json
        {
            "email": "XXXXXX@mac.com",
            "password": "XXXXXXXX",
            "platform": "myQ"
        }
```

**Screenshots:**

**Environment:**

* **Homebridge Version:**  1.1.7
* **Node Version:**  14.15.2
* **Homebridge-myQ Plugin Version**: 2.1.12
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iPhone 12, iOS 14.3
* **Operating System and OS Version:**  Raspbian 5.4.79-v7l+

**Comment by hjdhjd** [comment]
There have been API changes on the server end with myQ. You seem to be running an outdated version - the current release is v2.3.6. Give that a go! 😀

**Comment by DNSGeek** [comment]
That’s weird. The home bridge GUI shows all plugins up to date. How can I update it manually from the command line?

**Comment by donavanbecker** [comment]
@DNSGeek you should be able to update with the ui, try force restart with homebridge settings in the UI. or clicking on the reinstall button on the plugin.

**Comment by DNSGeek** [comment]
I restarted hoomebridge and rebooted the Pi. It’s still showing that homebridge-myq2’s latest version is 2.1.12

**Comment by DNSGeek** [comment]
Ok. I uninstalled and reinstalled the plugin and it found the new version. It’s working correctly now. Very strange that homebridge couldn’t see the new version. Thank you!

**Comment by dppeak** [comment]
There was a plugin name change from homebridge-myq2 to homebridge-myq that happened and that might be why you're not seeing the update.  2.1.12 was the version when that changed.  You need to uninstall myq2 and install myq.

[https://github.com/hjdhjd/homebridge-myq/tree/v2.1.12](https://github.com/hjdhjd/homebridge-myq/tree/v2.1.12)

**Comment by dppeak** [comment]
The config block remains the same, but the "platform" value changes to "myQ".

**Comment by Danielwinship** [comment]
Heads up in case anyone updates the plugin but still has the error. After installing the updated plugin I had to restart home bridge for the error to go away.

**Comment by hjdhjd** [comment]
You always need to restart Homebridge after you update any plugin. 😄

**Comment by jhays** [comment]
I've removed the older myq2 plugin, and installed myq 2.3.6.
(I ran "npm uninstall -g homebridge-myq2", it didn't seem to remove the node_module homebridge-myq2 folder, so I manually deleted that folder)

I've restarted homebridge, and tried restarting the pi it is running on. 

When homebridge boots, logging indicates:
===
Loaded plugin: homebridge-myq@2.3.6
Registering platform 'homebridge-myq.myQ'
===

So I'm pretty sure I'm on the right version.

I continue to get the error messages shown in the original post of this issue. 

I've also verified my credentials with the MyQ app- they still work.

Do I need to blow away persist or accessories folder contents in my homebridge directory? That's the only thing I haven't tried yet. 

Any other ideas?
Thanks
